internal/distro/rhel85/distro.go

```diff
@@ -163,7 +163,20 @@ func (t *imageType) Size(size uint64) uint64 {
 }
 
 func (t *imageType) PackageSets(bp blueprint.Blueprint) map[string]rpmmd.PackageSet {
-	return nil
+	sets := make(map[string]rpmmd.PackageSet)
+	for name, pkgSet := range t.packageSets {
+		if name == "packages" {
+			// combine image packages with blueprint
+			pkgSet.Include = append(pkgSet.Include, bp.GetPackages()...)
+			timezone, _ := bp.Customizations.GetTimezoneSettings()
+			if timezone != nil {
+				pkgSet.Include = append(pkgSet.Include, "chrony")
+			}
```

Comment:
Don't we also include the kernel here, or do I misremember?

Reply:
The kernel comes with the blueprint packages: https://github.com/osbuild/osbuild-composer/blob/cf73edd2d2248bbb5e561a8beeab694857be2132/internal/blueprint/blueprint.go#L107
including the fallback when unspecified:
https://github.com/osbuild/osbuild-composer/blob/cf73edd2d2248bbb5e561a8beeab694857be2132/internal/blueprint/customizations.go#L165-L167

Reply:
Ah, of course. I should know that ;)